src/localize-router.service.ts

```diff
@@ -60,8 +85,10 @@ export class LocalizeRouterService {
    * @returns {string}
    */
   private traverseRouteSnapshot(snapshot: ActivatedRouteSnapshot): string {
-    if (snapshot.firstChild && snapshot.firstChild.routeConfig && snapshot.firstChild.routeConfig.path) {
+    if (snapshot.firstChild && snapshot.firstChild.routeConfig && snapshot.firstChild.routeConfig.path && snapshot.firstChild.routeConfig.path !== '**') {
       return this.parseSegmentValue(snapshot) + '/' + this.traverseRouteSnapshot(snapshot.firstChild);
+    } else if (snapshot.firstChild && snapshot.firstChild.routeConfig && snapshot.firstChild.routeConfig.path && snapshot.firstChild.routeConfig.path === '**') {
```

Comment:
Please optimize this condition to do less checks e.g.:
if (snapshot.firstChild && snapshot.firstChild.routeConfig && snapshot.firstChild.routeConfig.path) {
  if ( snapshot.firstChild.routeConfig.path === '**') {
    return this.parseSegmentValue(snapshot.firstChild);
  }
  return this.parseSegmentValue(snapshot) + '/' + this.traverseRouteSnapshot(snapshot.firstChild);
}
return return this.parseSegmentValue(snapshot);